front/app/containers/Admin/projects/project/ideas/NewIdeaButtonDropdown.tsx

```diff
@@ -73,20 +73,16 @@ const NewIdeaButtonDropdown = ({
     <>
       <Box
         color="white"
-        background={colors.teal}
+        background={colors.primary}
         borderRadius="3px"
-        height="42px"
+        height="44px"
         px="16px"
+        py="2px"
         display="flex"
```

Comment:
We'll have to find another way here rather than using 2px to get alignment I think. Do you know what seems to be causing the issue? Could we add a property to the other button instead so they match? :)

Reply:
Just added a tweaked solution - removing all padding/margin from child text component instead

Reply:
Perfect - thanks!!